### DISCUSSION TITLE
I want to create a Solidworks to Freecad part file converter

### DISCUSSION THREAD

**Discussion by hemangjoshi37a** [FreeCAD/FreeCAD#18609] [Code contribution discussions]
and in the future also for the assembly files .

Please if anyone is interested we can colab on programming this .

I am thinking that this can be a web based UI that be ran locally and we can upload solidworks file and convert and download the freecad converted file. web based because if anyone want to host this on the web and share this with others that it is easily possible and also it is good for seo of this project.

Any kind of support in this direction is most welcome.

**Comment by luzpaz** [comment]
Talk to https://x.com/AleksanderSado1  
Maybe you can leverage https://www.fcviewer.com/ to do this?
